mapillary_tools/video_data_extraction/extractors/exiftool_runtime_parser.py

```diff
@@ -28,9 +29,14 @@ def __init__(
         self, video_path: Path, options: CliOptions, parser_options: CliParserOptions
     ):
         super().__init__(video_path, options, parser_options)
+        exiftool_path = shutil.which(constants.EXIFTOOL_PATH)
```

Comment:
Do we need this early check? I think `subprocess.run` will throw FileNotFound if its not there